pkg/reconciler/apis/apiexport/apiexport_reconcile.go

```diff
@@ -192,7 +192,9 @@ func (c *controller) updateVirtualWorkspaceURLs(apiExport *apisv1alpha1.APIExpor
 			apiExport.Name,
 		)
 
-		desiredURLs.Insert(u.String())
+		if !desiredURLs.Has(u.String()) {
+			desiredURLs.Insert(u.String())
+		}
```

Comment:
it's a set. You can insert multiple times

Reply:
okay, a set is implemented as a map